be/src/common/greplog.cpp

```diff
@@ -33,16 +33,17 @@ namespace starrocks {
 static std::vector<string> list_log_files_in_dir(const string& log_dir, char level) {
     std::vector<string> files;
     // if level in WARNING, ERROR, FATAL, use logging logs, else use info logs
-    const std::string pattern = string("WEF").find(level) == string::npos ? "be.INFO.log." : "be.WARNING.log.";
+    const std::string pattern = string("WEF").find(level) == string::npos ? ".INFO.log." : ".WARNING.log.";
```

Comment:
share define a global variable to indicate the current process is a compute node or backend node, and process the log in a more accurate way.

Reply:
done